src/installer/corehost/build.cmd

```diff
@@ -66,7 +66,7 @@ if "%VisualStudioVersion%"=="16.0" (
 :MissingVersion
 :: Can't find VS 2017, 2019
 echo Error: Visual Studio 2017 or 2019 required
```

Comment:
You can also delete 2017 from this list. We require 2019 now.

Reply:
Removed it from the message.
Left the functionality in the scripts that will still try to use 2017 - I think we can do that as a more involved change for #1251